plugins/colored-man-pages/colored-man-pages.plugin.sh

```diff
@@ -0,0 +1,47 @@
+#! bash oh-my-bash.module
+
+# Safety get the colors.
+if tput setaf 1 &> /dev/null; then
+  COLORED_MAN_PAGES_BLACK=$(tput setaf 0)
+  COLORED_MAN_PAGES_RED=$(tput setaf 1)
+  COLORED_MAN_PAGES_GREEN=$(tput setaf 2)
+  COLORED_MAN_PAGES_YELLOW=$(tput setaf 3)
+  COLORED_MAN_PAGES_BLUE=$(tput setaf 4)
+  COLORED_MAN_PAGES_MAGENTA=$(tput setaf 5)
+  COLORED_MAN_PAGES_CYAN=$(tput setaf 6)
+  COLORED_MAN_PAGES_WHITE=$(tput setaf 7)
+  COLORED_MAN_PAGES_BOLD=$(tput bold)
+  COLORED_MAN_PAGES_RESET=$(tput sgr0)
+
+  # Extra Styles
+  export LESS_TERMCAP_mr=$(tput rev) # Reverse
+  export LESS_TERMCAP_mh=$(tput dim) # Half Bright
+  export LESS_TERMCAP_ZN=$(tput ssubm) # Subscript
+  export LESS_TERMCAP_ZV=$(tput rsubm) # Reverse Superscript
+  export LESS_TERMCAP_ZO=$(tput ssupm) # Subscript
+  export LESS_TERMCAP_ZW=$(tput rsupm) # Reverse Superscript
```

Comment:
`tput` doesn't necessarily recognize the terminfo names. In some (non-Linux) systems, `tput` only accepts termcap names. For such a case, could you redirect `stderr` and also attempt a call with the termcap name? You can find examples in

https://github.com/ohmybash/oh-my-bash/blob/f53beb64b8fe8a3f6ad9df09cc42fa5570406def/lib/utils.sh#L150-L160

Reply:
Didn't know of those. Will update accordingly. Thanks.